compiler+runtime/src/cpp/jank/runtime/context.cpp

```diff
@@ -305,8 +307,7 @@ namespace jank::runtime
 
     binding_scope const preserve{ *this,
                                   obj::persistent_hash_map::create_unique(
-                                    std::make_pair(compile_files_var, jank_true),
-                                    std::make_pair(current_module_var, make_box(module))) };
+                                    std::make_pair(compile_files_var, jank_true)) };
```

Comment:
What's the purpose of this change?

Reply:
Here, I removed the `current_module_var` because `load_module` will set the `current_module_var`.